src/gameLogic/generation/actions/GambleActions.cpp

```diff
@@ -23,11 +22,10 @@ namespace spy::gameplay {
                                                                 });
 
         for (auto &p: points.first) {
```

Comment:
Hier wird p nie editiert, kann also const sein.

```suggestion
        for (const auto &p: points.first) {
```